src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/custom.py

```diff
@@ -15,12 +15,15 @@
     ConsistencyPolicy,
     DatabaseAccountCreateUpdateParameters,
 )
+from azure.mgmt.documentdb.models.document_db_enums import DatabaseAccountKind
+
 
 # pylint:disable=too-many-arguments
 def cli_documentdb_create(client,
                           resource_group_name,
                           account_name,
                           locations,
+                          kind="GlobalDocumentDB",
```

Comment:
Get this value from the SDK enum instead of hardcoding it here.
`DatabaseAccountKind....`